pytest_httpserver/httpserver.py

```diff
@@ -452,8 +467,17 @@ def respond(self, request: Request) -> Response:
 
         if self.request_handler is None:
             raise NoHandlerError("No handler found for request: {} {}".format(request.method, request.path))
-        else:
-            return self.request_handler(request)
+        self.calls.append(request)
+        return self.request_handler(request)
+
+    def calls_data(self, auto_load: bool = True):
```

Comment:
I have problem with the naming, I expect that calls_data returns bytes, not arbitrary objects. There could be calls_json but we could implement objects which have json, data, and other methods/properties similar to my code in the other PR.

Having:
```
httpserver.calls.json # returns json loaded
```
and
```
httpserver.calls.data # returns bytes
```

Makes it clear what the user wants to see, while also keeping the attributes of the object at low by not adding `calls_json`, and `calls_data` to it but only `calls`.

While having `calls_data(True)` says nothing about it, and also `calls_data(auto_load=True)`.